src/pages/ImportData.js

```diff
@@ -107,6 +123,7 @@ const ImportData = () => {
   )(async workspace => {
     const namespace = workspace.namespace
     const name = workspace.name
```

Comment:
While you're in here, could you just
```suggestion
    const { namespace, name } = workspace
```